src/hooks/useAutoHeight.ts

```diff
@@ -17,7 +21,9 @@ export function useAutoHeight() {
     const offsetBasedHeight = `calc(100vh - ${topOffset}px - 24px)`;
 
     const parentHeight = containerRef.parentElement?.clientHeight;
-    const parentPadding = containerRef.parentElement?.style.paddingBottom;
+    const parentPadding = containerRef?.parentElement?.style.paddingBottom
```

Comment:
This doesn't need to be a ternary since the conditional is the exact same as the truthy evaluation - can just do `const parentPadding = containerRef?.parentElement?.style.paddingBottom || '0px';`